chalicelib/checks/system_checks.py

```diff
@@ -13,29 +12,51 @@
 )
 
 import requests
-import sys
 import json
 import datetime
 import boto3
 import time
 
 
-@check_function()
-def wipe_build_indices(connection, **kwargs):
-    """ Wipes all indices on the FF-Build ES env """
+# XXX: put into utils?
+CGAP_TEST_CLUSTER = 'search-cgap-testing-ud3ggpjj7x6vclx62nmzymyzfi.us-east-1.es.amazonaws.com:80'
+FF_TEST_CLUSTER = 'search-fourfront-testing-hjozudm7pq5wwlssx2pjlsyz7y.us-east-1.es.amazonaws.com:80'
+TEST_ES_CLUSTERS = [
+    CGAP_TEST_CLUSTER,
+    FF_TEST_CLUSTER
+]
+
+
+def wipe_build_indices(connection, es_url):
```

Comment:
The argument is not really a `URL`.  I would call it an `es_cluster` or `es_server` or `es_host` or `es_host_and_port`. Although maybe it's elsewhere called a URL and you're just going for consistency...Is there an `ES_URL` config param? Alas. Whatever you think is reasonable. I just wanted to mention the issue.

Reply:
I chose `es_url` to be consistent with: https://github.com/4dn-dcic/utils/blob/master/dcicutils/es_utils.py#L11 but it is not a "standard" so to speak.